pkg/netpol/connlist/connlist.go

```diff
@@ -76,6 +77,12 @@ func WithFocusWorkload(workload string) ConnlistAnalyzerOption {
 	}
 }
 
+func WithOutputFormat(outputFormat ConnsFormatter) ConnlistAnalyzerOption {
+	return func(p *ConnlistAnalyzer) {
+		p.outputFormat = outputFormat
+	}
+}
+
```

Comment:
since this is the interface to be used also as a library from external users, I prefer that the `outputFormat` type here is `string` and converted to `ConnsFormatter` internally. Also `ConnsFormatter` can be internal to this package and not exported.

Reply:
done